userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -184,6 +184,11 @@ To upgrade to a specific AMI release version instead of the latest version, pass
 eksctl upgrade nodegroup --name=managed-ng-1 --cluster=managed-cluster --release-version=1.19.6-20210310
 ```
 
+## Updating managed nodegroups
+It is also possible to update specific fields of a managed nodegroup with the command `eksctl update nodegroup` without using `upgrade`. This new command currently only supports a few fields and is intended to be used as a way to modify the configuration of a nodegroup without triggering an entire upgrade.
```

Comment:
shall we add a list of the fields? or I guess in this PR there isn't any yet?

Reply:
I had added a link to the example config but linter didn't like it because the link will appear broken until this PR is merged! https://github.com/weaveworks/eksctl/pull/3828/commits/4d888bc40944257b1a8e9bbbdb7a02779f262292

Reply:
I've made a note to make a one-liner PR after this one is merged to add the link to the example again, so that the linter doesn't complain :)

Reply:
Can we add more to make it very clear that this will only ever update upgrade configuration?

I mean, who knows what we may use it for later, but let's not lead anyone on.

Reply:
definitely, more info will be added in the PR that joins `update nodegroup` and `updateConfig`